### PR TITLE
Add view settings to milestones-editor

### PR DESCRIPTION
**Pull request by IidaKainu**
Settings are saved to localStorage by setting string and current userId and roadmapId.

### CODE REVIEW

File: frontend/src/pages/MilestonesEditor.module.scss [hunk 1]
```diff
@@ -146,7 +147,6 @@
 .topbar {
   @extend .layout-row;
   min-height: 60px;
-  overflow-x: auto;
```
Comment (IidaKainu) [inline frontend/src/pages/MilestonesEditor.module.scss:149]: This sadly brings back the double-scrollbar on the bottom of the page. 
With the line, the menu content would not pop over elements (instead locating itself inside the top bar, in a vertical scrollbox)

Comment (PatricKangasmaki) [reply frontend/src/pages/MilestonesEditor.module.scss:149]: Well, I guess we can live with that. The double-scroll bar becomes visible only when your screen with is under 1035px, so it shouldn't be there too often and even when it's there, it's nothing too bad. If you'd like to fix it, you could for example make the navbar go on two lines (if possible) or hide certain not-so-important-things there, such as the ones in the image, when going under certain width thresholds. Or make the weighted shares bar narrower etc. But IMO it's okay to leave as it is as well

image

**Review by PatricKangasmaki — APPROVED** [review]
Looking and working great 👍 I'll approve this now and leave the double-scrollbar up to you if you have time to try to fix it or not

File: frontend/src/pages/MilestonesEditor.module.scss (hunk 1 above)
Comment (IidaKainu) [reply frontend/src/pages/MilestonesEditor.module.scss:149]: Good suggestions, but I'll leave it as is due to the time constraints

**Pull request merged**